blue-green-deploy/README.md

```diff
@@ -0,0 +1,90 @@
+# Blue Green Deploy on Kubernetes
+
+## Introduction
+
+One of the challenges with automating deployment is the cut-over itself, taking software from the final stage of testing to live production. You usually need to do this quickly in order to minimize downtime. The blue-green deployment approach does this by ensuring you have two production environments, as identical as possible. At any time one of them, let's say blue for the example, is live. As you prepare a new release of your software you do your final stage of testing in the green environment. Once the software is working in the green environment, you switch the router so that all incoming requests go to the green environment - the blue one is now idle.
+
+Blue-green deployment also gives you a rapid way to rollback - if anything goes wrong you switch the router back to your blue environment.
```

Comment:
So we have the "marketting" description on how blue green deployment works but it would be better i think to explains how the task actually works? I would not know how it works unless i start to read the shell script....